lib/games.rb

```diff
@@ -0,0 +1,35 @@
+class Games
```

Comment:
These classes will help us keep our stat tracker calls small and compact, as well as homogenize our header names.